lms/djangoapps/instructor_task/api.py

```diff
@@ -114,6 +115,28 @@ def submit_rescore_problem_for_student(request, usage_key, student, only_if_high
     return submit_task(request, task_type, task_class, usage_key.course_key, task_input, task_key)
 
 
+# Disabling invalid-name because this fn name is longer than 30 chars.
+def submit_override_problem_score_for_student(request, usage_key, student, score):  # pylint: disable=invalid-name
+    """
+    Request a problem score override as a background task.
+
+    The problem score will be overridden for the specified student only.
+    Parameters are the `course_id`, the `problem_url`, the `student` as
+    a User object, and the score override desired.
+    The url must specify the location of the problem, using i4x-type notation.
+
+    ItemNotFoundException is raised if the problem doesn't exist, or AlreadyRunningError
+    if this task is already running for this student, or NotImplementedError if
+    the problem is not a ScorableXBlock.
+    """
+
+    task_type = 'override_problem_score'
```

Comment:
Note that the `task_type` is just the name of the `task_class`, which can be gotten with `override_problem_score.__name__`.